x-pack/plugins/fleet/server/services/epm/archive/storage.ts

```diff
@@ -20,26 +18,22 @@ import type {
   InstallablePackage,
   InstallSource,
   PackageAssetReference,
-  RegistryDataStream,
 } from '../../../../common/types';
-import { pkgToPkgKey } from '../registry';
 
 import { appContextService } from '../../app_context';
 
 import { getArchiveEntry, setArchiveEntry, setArchiveFilelist, setPackageInfo } from '.';
 import type { ArchiveEntry } from '.';
-import {
-  parseAndVerifyPolicyTemplates,
-  parseAndVerifyStreams,
-  parseTopLevelElasticsearchEntry,
-} from './parse';
+import { parseAndVerifyArchive } from './parse';
 
 const ONE_BYTE = 1024 * 1024;
 // could be anything, picked this from https://github.com/elastic/elastic-agent-client/issues/17
 const MAX_ES_ASSET_BYTES = 4 * ONE_BYTE;
 // Updated to accomodate larger package size in some ML model packages
 const ML_MAX_ES_ASSET_BYTES = 50 * ONE_BYTE;
 
+const MANIFEST_NAME = 'manifest.yml';
```

Comment:
nit: this const could be exported from [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/epm/archive/parse.ts#L41) rather than duplicating